gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
+#Fri Apr 26 13:57:56 MST 2019
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.3-bin.zip
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.3-all.zip
```

Comment:
❓What is the difference between `gradle-4.10.3-bin` and `gradle-4.10.3-all`?

Reply:
Follow-on question: Do we have any reason to not use the latest version: Gradle 5.x.x?

Reply:
Both are questions I don't know the answer to. We should switch to 5.x.x before the gradle file becomes tech debt